app/src/main/res/layout/fragment_home.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+    <data>
+        <import type="android.view.View" />
+        <variable
+            name="viewModel"
+            type="com.sanjeevyadavit.magecart.viewmodel.MainActivityViewModel" />
+    </data>
```

Comment:
You can try doing these things in the xmls where you are having dynamic data. o.e user input or things like that.

Reply:
You do not need viewModel in this. You may need later. but currently you are not using viewModel's data in this xml file.